src/plugins/opensearch_dashboards_overview/public/plugin.ts

```diff
@@ -81,9 +81,17 @@ export class OpenSearchDashboardsOverviewPlugin
       map((hasOpenSearchDashboardsApp) => {
         return () => {
           if (!hasOpenSearchDashboardsApp) {
-            return { status: AppStatus.inaccessible, navLinkStatus: AppNavLinkStatus.hidden };
+            return {
+              status: AppStatus.inaccessible,
+              navLinkStatus: AppNavLinkStatus.hidden,
+              workspaceAccessibility: WorkspaceAccessibility.NO,
```

Comment:
Change on registeration is enough I think.

Reply:
Have deleted the useless code.